src/components/IssueUI/IssueRequestStatusUI/ManualIssueExecutionUI/index.tsx

```diff
@@ -75,8 +75,7 @@ const ManualIssueExecutionUI = ({ request }: Props): JSX.Element => {
       const issue = await window.bridge.issue.getRequestById(request.id);
 
       const collateralToken = await currencyIdToMonetaryCurrency(
-        window.bridge.assetRegistry,
```

Comment:
Has this been tested for regressions?

Reply:
This has not been tested for regression

Reply:
Ok, no worries—can you make a note that we'll need to test it once the code has been merged? Doesn't need to hold up the AMM testnet release.

Reply:
@peterslany can answer better, but let's check it out once it is in testnet

Reply:
> @peterslany can answer better, but let's check it out once it is in testnet

There was change to this function's signature and function was extended with LP token types. I think typechecking from TSC made sure that it is refactored in all places correctly. However, there were some additionally changes to how currencies are handled on UI afaik(right @danielsimao ?), so I would prefer testing it :+1:

Reply:
Thanks @peterslany — if you can make a note of this, we'll make sure it's tested ahead of releasing AMM to mainnet. Doesn't need to hold up the testnet release though.